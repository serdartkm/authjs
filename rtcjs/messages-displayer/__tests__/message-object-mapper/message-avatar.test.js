import React from 'react'
import MessageAvatar from '../../lib/message-object-mapper/message-avatar'
import renderer from 'react-test-renderer'


describe("SNAPSHOT TEST MessageAvatar",()=>{

    it("MessageAvatar renders correctly",()=>{
        const tree = renderer.create(<MessageAvatar letter="A"/>).toJSON()

        expect(tree).toMatchSnapshot();

    })
})