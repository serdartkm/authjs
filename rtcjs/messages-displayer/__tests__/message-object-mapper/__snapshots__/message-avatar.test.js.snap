// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNAPSHOT TEST MessageAvatar MessageAvatar renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "darkSmoke",
      "borderColor": "#80cbc4",
      "borderRadius": 30,
      "borderStyle": "solid",
      "borderWidth": 2,
      "color": "#009688",
      "display": "flex",
      "height": 40,
      "justifyContent": "center",
      "padding": 3,
      "width": 40,
    }
  }
>
  <div>
    A
  </div>
</div>
`;
